docs/user-guide/templates.rst

```diff
@@ -0,0 +1,77 @@
+.. _templates:
+
+Using custom templates
+----------------------
+
+c2cgeoform distinguishes between two types of templates: **views** templates
+and **widget** templates.
+- Views templates are used directly by Pyramid and provide the site structure.
+- Widgets templates are used by Deform to render the forms.
+
+Default views templates
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The default c2cgeoform views templates are located in the ``templates``
+folder and use `jinja2`_ syntax.
+
+c2cgeoform comes with partial templates that are included in views templates
+of your project.
+
+.. _Jinja2: http://jinja.pocoo.org/
+
+Overriding widgets templates globally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Deform widget templates are located in the ``templates/widgets`` folder and
+use the `chameleon`_ syntax.
+
+At rendering time, Deform will search for the templates as configured in Form
```

Comment:
At rendering time, Deform will search for the templates that are configured in Form